spark/src/main/scala/com/cloudera/livy/spark/client/ClientSession.scala

```diff
@@ -19,31 +19,46 @@ package com.cloudera.livy.spark.client
 import java.net.URI
 import java.nio.ByteBuffer
 import java.util.concurrent
-import java.util.concurrent.{ExecutionException, TimeUnit}
+import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicLong
 
+import org.apache.spark.SparkConf
```

Comment:
We should standardize on one import order... I've been following Spark's style, but the rest of the code is a mess of different styles.

Using Spark's style, this import is in the wrong place.